conda/resolve.py

```diff
@@ -930,7 +930,7 @@ def version_key(self, prec, vtype=None):
         channel = prec.channel
         channel_priority = self._channel_priorities_map.get(
             channel.name, 1
-        )  # TODO: ask @mcg1969 why the default value is 1 here  # NOQA
+        )  # TODO: ask @mcg1969 why the default value is 1 here
```

Comment:
@mcg1969 care to comment here or can we remove this comment?

Reply:
@kenodegard there is the FIX ruleset to address those TODO lines, see https://docs.astral.sh/ruff/rules/#flake8-fixme-fix ... that goes beyond the intend of this PR.